src/js/views/admin/program/flow/action-item.hbs

```diff
@@ -0,0 +1,21 @@
+<td class="table-list__cell w-40">
```

Comment:
This might not be ideal, given we are trying to dedupe our codebase. But this template contains a wrapping div within the meta `td` so that we can absolutely position the added `program-flow__action-sort` element. We could conditionally add both of those things to the generic `action-item.hbs` using an `isSortable` flag. @paulfalgout do you have a preference?

Reply:
alrighty, put up #242 to revert #240, in the interest of getting this finished quicker. We can readdress that down the line once this has settled